lambda/custom/index.js

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable  no-console */
 
 const Alexa = require('ask-sdk-core');
-const functions_axios = require('./functions_axios');
+const functionsAxios = require('./functionsAxios');
```

Comment:
Change to helperFunctions. Avoid using axiosFunctions as that file can have all the extra functions we need which may or may not require axios.